Source/UI/NewUI/BillingForm/View/TableViewCell/Header/BillingFormHeaderCell.swift

```diff
@@ -0,0 +1,138 @@
+import UIKit
+
+protocol BillingFormHeaderCellDelegate: AnyObject {
+    func doneButtonIsPressed()
+    func cancelButtonIsPressed()
+}
+
+final class BillingFormHeaderCell: UITableViewCell {
+    private var style: BillingFormHeaderCellStyle
+    private weak var delegate: BillingFormHeaderCellDelegate?
+    
+    private lazy var cancelButton: UIButton = {
+        let view = UIButton()
+        view.setTitle(style.cancelButton.text, for: .normal)
+        view.titleLabel?.font = style.cancelButton.font
+        view.setTitleColor(style.cancelButton.activeTitleColor, for: .normal)
+        view.setTitleColor(style.cancelButton.inActiveTitleColor, for: .disabled)
+        view.tintColor = view.isEnabled ? style.cancelButton.activeTintColor : style.cancelButton.inActiveTintColor
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.backgroundColor = style.cancelButton.backgroundColor
+        view.addTarget(self, action: #selector(cancelAction), for: .touchUpInside)
+        return view
+    }()
+    
+    private lazy var done: UIButton = {
+        let view = UIButton()
+        view.isEnabled = style.doneButton.isEnabled
+        view.setTitle(style.doneButton.text, for: .normal)
+        view.titleLabel?.font = style.doneButton.font
+        view.setTitleColor(style.doneButton.activeTitleColor, for: .normal)
+        view.setTitleColor(style.doneButton.inActiveTitleColor, for: .disabled)
+        view.tintColor = view.isEnabled ? style.doneButton.activeTintColor : style.doneButton.inActiveTintColor
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.backgroundColor = style.doneButton.backgroundColor
+        view.addTarget(self, action: #selector(doneAction), for: .touchUpInside)
+        return view
+    }()
+    
+    private lazy var headerLabel: UILabel = {
```

Comment:
I can't see this field used outside of the creation and setup, so I wonder if we need to have this property, or when we create it for the view, we just create, customise, add to view and allow to lose local reference.

Probably other similar UI components are doing the same thing?

Reply:
Resolved on slack